scripts/zones/Jugner_Forest/mobs/Fradubio.lua

```diff
@@ -2,13 +2,78 @@
 -- Area: Jugner_Forest
 --   NM: Fradubio
 -----------------------------------
+local ID = require("scripts/zones/Jugner_Forest/IDs")
+-----------------------------------
 local entity = {}
 
+local updateRegen = function(mob)
+    local hour = VanadielHour()
+    if hour >= 4 and hour < 20 then
+        mob:setMod(xi.mod.REGEN, 25)
+    else
+        mob:setMod(xi.mod.REGEN, 0)
+    end
+end
+
+entity.onMobSpawn = function(mob)
+    updateRegen(mob)
+end
+
+entity.onMobFight = function(mob)
+    updateRegen(mob)
+end
+
+entity.onMobRoam = function(mob)
+    updateRegen(mob)
+end
+
+entity.onMobWeaponSkill = function(target, mob, skill)
+    if skill:getID() == 329 then
+        for i = ID.mob.DUESSA_START, ID.mob.DUESSA_END do
+            local pet = GetMobByID(i)
+            if not pet:isSpawned() then
+                local petId = pet:getID()
+                switch (petId): caseof {
+                    [17203449] = function()
```

Comment:
we definitely do not want magic numbers in script

Reply:
mob AI should automatically try and spread these out. if it doesn't I feel that's deserving of an issue.

Reply:
Took out that code, it worked fine without it.